.setup/data/users/instructor.yml

```diff
@@ -3,5 +3,6 @@ user_group: 1
 user_id: instructor
 anon_id: lokMH456u2As096
 user_familyname: Instructor
+user_pronouns: She/Her
```

Comment:
I forgot to add comment the discussion we had during meeting, order should not matter, but would be ideal to have consistent user_pronoun label to always be below user_numeric_id. However, if it too much of a burden please feel free to mark it as resolved.